src/automations/index.js

```diff
@@ -23,6 +23,21 @@ export default class Automations {
     })
   }
 
+  async activateFlowById(flowId, initialData) {
+    if (!flowId || typeof flowId !== 'string') {
+      throw new Error('The "flowId" argument must be provided and must be a string.')
+    }
+
+    if (initialData !== undefined && !Utils.isObject(initialData)) {
+      throw new Error('The "initialData" argument must be an object with an arbitrary structure.')
```

Comment:
Just `The "initialData" argument must be an object`, no need to require this object to be an arbitrary structure.

Reply:
\+